website/docs/docs/build/incremental-models.md

```diff
@@ -450,5 +450,78 @@ The syntax depends on how you configure your `incremental_strategy`:
 
 </VersionBlock>
 
+### Custom strategies
+
+<VersionBlock lastVersion="1.1">
+
+Custom incremental strategies can be defined beginning in dbt v1.2.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.2">
+
+As an easier alternative to [creating an entirely new materialization](/guides/create-new-materializations), users can define and use their own "custom" user-defined incremental strategies by:
+
+1. defining a macro named `get_incremental_STRATEGY_sql`
```

Comment:
are we saying the macro should always be named `get_incremental_STRATEGY_sql`?

Reply:
or is `STRATEGY` a placeholder? if its a placeholder, we should maybe be more clear on that.

Reply:
Yep, `STRATEGY` is a placeholder!

Any ideas how to make it more clear?

Along the lines of your suggestion about the table, you think adding adding it would make it more clear?

![Screenshot 2024-01-08 at 12 10 43](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/f5bff8cc-1ab8-4391-9748-fb3168817945)

Reply:
```
users can define and use their own "custom" user-defined incremental strategies by:

1. defining a macro named `get_incremental_STRATEGY_sql`. Note that `STRATEGY` is a placeholder and you should replace it with the name of your custom incremental strategy.
2. configuring `incremental_strategy: STRATEGY` within an incremental model

dbt won't validate user-defined strategies, it will just look for the macro by that name, and raise an error if it can't find one.

For example,.....

```

Reply:
ok @dbeatty10 proposing the above adn before i make the changes or sugg - please let me know if that makes sense to you 🙏

Reply:
This looks good 🚀

Reply:
ok cool, will make that change!